ui/ducks/metamask/metamask.js

```diff
@@ -398,6 +399,10 @@ export const getGasFeeEstimates = createSelector(
   },
 );
 
+export const getSmartTransactionFees = (state) => {
```

Comment:
Rather than adding new selectors to this legacy file, could we create a new `selectors/smart-transactions.ts` or even put it in the shared selectors?